### ISSUE TITLE
Esri Datenaufbereitung

### ISSUE THREAD

**Issue by zdavatz** [daenuprobst/covid19-cases-switzerland#13]
Es wäre auch gut, wenn die Daten in diesem Format (gleich wie Italien und China) aufbereitet werden:

Pro Tag eine Zeile für alle Kantone
https://github.com/zdavatz/covid19_ch/blob/master/data-cantons-csv/dd-covid19-ch-cantons-20200318-example.csv

Pro Tag eine Zeile für die ganze Schweiz.
https://github.com/zdavatz/covid19_ch/blob/master/data-switzerland-csv/dd-covid19-ch-switzerland-20200318-example.csv

**Comment by zdavatz** [comment]
New sample files are now partially filled.

daily Cantons file:
https://github.com/zdavatz/covid19_ch/blob/master/data-cantons-csv/dd-covid19-ch-cantons-20200319-example.csv

Total file:
https://github.com/zdavatz/covid19_ch/blob/master/data-switzerland-csv/dd-covid19-ch-switzerland-20200319-example.csv

**Comment by tgdbepe4** [comment]
Würde dies auch sehr begrüssen, weil dann eine Datenvisualisierung viel einfacher würde. Hat sich diesbezüglich schon was getan?

**Comment by zdavatz** [comment]
@tgdbepe4 Ja, viel hat sich getan, siehe https://github.com/zdavatz/covid19_ch falls Du mithelfen willst. Wir haben auch ein Python-Script, siehe: https://github.com/zdavatz/covid19_ch/tree/master/python-scripts

**Comment by zdavatz** [comment]
https://github.com/daenuprobst/covid19-cases-switzerland/blob/master/covid_19_cases_switzerland_standard_format.csv
